### DISCUSSION TITLE
How can I see the prioritisation of upcoming branches?

### DISCUSSION THREAD

**Discussion by allanlewis** [renovatebot/renovate#28978] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

I set `prPriority` for some package groups but it's not clear to me what the overall effect of these is. How can I get a list of upcoming branches in priority order, or something similar that will help me tweak priorities?

I can run Renovate locally using `--dry-run` to test things - I find the `--report-path` option particularly helpful.

### Logs (if relevant)

n/a

**Comment by rarkins** [comment, marked as answer]
Isn't it reflected in the order in Dependency Dashboard?

**Reply by allanlewis** [reply]
I recall being told that wasn't the case in a previous query - anyway, if it is then that's great! However, the dashboard order seems to ignore scheduling. For example, I have set lock file maintenance to have a high priority but a monthly schedule, so it appears at the top even when it isn't due for several weeks.

**Reply by rarkins** [reply]
I don't think we'll be able to "predict" the next PR to be created if it's also based on scheduling

**Reply by allanlewis** [reply]
Perhaps branches with a custom schedule could be marked with a 📆 icon or similar?

Anyway, if the list is in priority order then that's great 🙂

**Reply by rarkins** [reply]
Those with a custom scheduled should be under the awaiting schedule category

**Reply by allanlewis** [reply]
I find that's the case for most branches but not for lock-file maintenance.
